src/planscape/forsys/rank_projects_for_a_single_scenario.R

```diff
@@ -0,0 +1,44 @@
+suppressMessages({
+  library(dplyr)
+  library(forsys)
+  library(sf)
+})
+
+rank_projects_for_a_single_scenario <- function(input_stand_data,
+                                                priorities,
+                                                priority_weights,
+                                                stand_id_field,
+                                                proj_id_field,
+                                                stand_area_field,
+                                                stand_cost_field) {
+
+  scenario_output_fields <- c(priorities, stand_area_field, stand_cost_field)
```

Comment:
can we define this in-line in forsys::run parameters?

Reply:
I don't think it's necessary to make this an input parameter because 1) it can be computed from the given inputs, and 2) it's consistent.
If either of the reasons becomes untrue, we can return to this.

Reply:
oh maybe we're talking about different things, i meant an in-line input to the function called below, not the function this line is in. what do you mean when you say it's consistent?

Reply:
I see. done.

It's consistent in that there are no cases where we might want c(stand_area_field, stand_cost_field) instead of c(priorities, stand_area_field, stand_cost_field).